user-guide/Getting_started/DaaS/Order_DataMiner_credits.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: Order_DataMiner_credits
+---
+
+# Order DataMiner credits
+
+1. Go to the [Azure Marketplace](https://azuremarketplace.microsoft.com/) and Search for DataMiner.
+    ![Azure Marketplace DataMiner credits offer](~/user-guide/images/Azure_Marketplace_Offer.png)
+
+1. Click on *Get It Now* and continue to Azure portal.
+
+> [!WARNING]
+> make sure you have selected the plan Dataminer credits.
+
+1. Once redirected to the Azure portal click on Subscribe. This will start the process of creating a SaaS subscription resource to be deployed in a resource group.
```

Comment:
```suggestion
1. Once redirected to the Azure portal click on *Subscribe*. This will start the process of creating a SaaS subscription resource to be deployed in a resource group.
```